TODO_MANAGER_FILE

```diff
@@ -0,0 +1,209 @@
+import mysql.connector
```

Comment:
1. Create a folder "src" at root directory of the project
2. Place this file inside the "src" folder
3. Rename this file to todo_manager.py

Reply:
changes done